obspy-test-sc3ml_response.py

```diff
@@ -0,0 +1,3 @@
+from obspy.io.seiscomp.inventory import _read_sc3ml
+inv=_read_sc3ml('IM.I31KZ_SC3_formatted.xml')
+print(inv)
```

Comment:
This needs to be refactored into a proper unittest method in `obspy/io/seiscomp/tests/test_inventory.py`

Reply:
A test is now added to `obspy/io/seiscomp/tests/test_inventory.py`:

```
    def test_omit_response(self):
        """
        Test SC3ML inventory with response information that leads to ObspyException
        Read up to channel level and omit response information
        """
        inv = read_inventory(os.path.join(self.data_dir,
                                          "IM.I31KZ.sc3ml"))
        self.assertEqual(inv[0].code, "IM")
        self.assertEqual(inv[0][0].code, "I31H1")
        for cha in inv[0][0].channels:
            self.assertTrue(cha.code in ["BDF", "LWD", "LWS", "LKO", "LDA"])

```

Reply:
By the way, I noticed that the  test  `def test_stage_empty_poles_and_zeros(self):` gave errors, but it also seems to give errors in the obspy/master branch...